docs/topics/api/promoted.rst

```diff
@@ -0,0 +1,21 @@
+================
+Promoted add-ons
+================
+
+.. note::
+    These APIs are subject to change at any time and are for internal use only.
+
+
+--------------
+Stripe Webhook
+--------------
+
+.. _stripe-webhook:
+
+This endpoint receives `event notifications
+<https://stripe.com/docs/webhooks>`_ from Stripe.
+
+    .. note::
+        Requests are signed by Stripe and verified by the server.
+
+.. http:get:: /api/v4/promoted/stripe-webhook
```

Comment:
is there no standard about what's in the payload we can document?

Reply:
Probably documented somewhere in the Stripe docs. It's not like we can fake such requests, we need to use their CLI, which can forward events locally, simulate all kind of notifications, etc. I don't think we should document anything else given that we cannot create such requests manually.